pkg/engine/internal/planner/logical/planner_test.go

```diff
@@ -123,18 +123,19 @@ RETURN %21
 }
 
 func TestConvertAST_MetricQuery_Success(t *testing.T) {
-	q := &query{
-		statement: `sum by (level) (count_over_time({cluster="prod", namespace=~"loki-.*"} |= "metric.go"[5m]))`,
-		start:     3600,
-		end:       7200,
-		interval:  5 * time.Minute,
-	}
+	t.Run(`sum by (level) (count_over_time({cluster="prod", namespace=~"loki-.*"} |= "metric.go"[5m]))`, func(t *testing.T) {
```

Comment:
nit: make name descriptive

```suggestion
	t.Run("simple metric query", func(t *testing.T) {
```

Reply:
fixed